foreman/data_refinery_foreman/surveyor/management/commands/start.py

```diff
@@ -1,4 +1,4 @@
-from django.core.management.base import BaseCommand, CommandError
+from django.core.management.base import BaseCommand
```

Comment:
Ah ok - you might have mentioned this somewhere else, but any reason why we are not using Django's test suite to run the tests?